src/objc/abstract/MCOConstants.h

```diff
@@ -413,6 +413,10 @@ typedef NS_ENUM(NSInteger, MCOErrorCode) {
     MCOErrorNoValidServerFound,
     /** Error while running custom command */
     MCOErrorCustomCommand,
+    /** Spam was suspected by server */
+    MCOErrorSendMessageSpamSuspected,
```

Comment:
Could you prefix those error messages with Yahoo?

See:
MCOErrorGmailTooManySimultaneousConnections
or
MCOErrorYahooUnavailable